python/orca/tutorial/xshards/notebooks/auto_mpg.ipynb

```diff
@@ -797,24 +798,14 @@
     "    return model"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "ac0db7e4",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "model = build_model()"
```

Comment:
Using `build_model()` as the model creator function.
This code block is not used, thus removed. This is due to hamster not due to vscode 😢